pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -307,11 +307,15 @@ func (policy *regoEnforcer) getReasonNotAllowed(enforcementPoint string, input i
 	if err == nil {
 		errors, _ := result.Value("errors")
 		if errors != nil {
-			return fmt.Errorf("%s not allowed by policy. Errors: %v.\nInput: %s", enforcementPoint, errors, string(inputJSON))
+			if len(errors.([]interface{})) > 0 {
```

Comment:
to avoid potential panic, lets check that the type is correct
```
if e, ok := errors.([]interface{}); ok && len(e) > 0 {
    return fmt.Errorf("%s not allowed by policy. Errors %v. Input: %s", enforcementPoint, e, string(inputJSON))
}
```
Not sure if we need to handle incorrect return type in any special way though, but returning the below error should probably be fine.

Reply:
and if it isn't, what are we going to do? we can't not do something...

there's literally no way at the moment for this not to be that interface, so what is the correct thing to do? there's a message that needs to be displayed but the else block also means "the world is on fire".

Given that this needs to be in for monday, to expedite, are you ok with making whatever change you want to see here and handling the `else` case however you see fit?